server/src/services/apps.service.ts

```diff
@@ -290,6 +290,18 @@ export class AppsService {
   }
 
   async createVersion(user: User, app: App, versionName: string, versionFromId: string): Promise<AppVersion> {
+    const isEmpty = !versionName.length;
+
+    if (isEmpty) {
+      throw new BadRequestException('The version name should not be empty');
+    }
+
+    const isWithinCharacterLimit = versionName.length <= 255;
```

Comment:
You can rather use validation on _version-create.dto.ts_ file with a decorator like:
```
@MaxLength(255, {
    message: 'The version name cannot be more than 255 characters',
  })
```